lib/feedjira/date_time_utilities/date_time_epoch_parser.rb

```diff
@@ -0,0 +1,13 @@
+# rubocop:disable Style/Documentation
+# rubocop:disable Style/DocumentationMethod
+module Feedjira
+  module DateTimeUtilities
+    class DateTimeEpochParser
+      def self.parse(string)
+        epoch_time = string.to_i
+        return Time.at(epoch_time).to_datetime if epoch_time.to_s == string
+        raise "#{string} is not a valid epoch time"
```

Comment:
If I was going to refactor this code, I’d try to make it not use exceptions for control flow. I think that would be worth doing, but I prefer to keep my refactorings separate from my new features.